react-components/src/data-providers/core-dm-provider/utils/filters.ts

```diff
@@ -46,3 +47,9 @@ export function getRevisionContainsAnyFilter(
     }
   } as const satisfies TableExpressionContainsAnyFilterV3;
 }
+
+export function hasIdFilter(refs: DmsUniqueIdentifier[]): TableExpressionLeafFilter {
+  return {
+    instanceReferences: refs
+  } as unknown as TableExpressionLeafFilter;
```

Comment:
Remove `as unknown as X`.

I see this gives the error:
`Object literal may only specify known properties, and 'instanceReferences' does not exist in type 'TableExpressionLeafFilter'.`

Reply:
That is because this query type is not yet reflected in the SDK types

Reply:
Let's add it do the SDK?

Reply:
This will likely be added in this PR: https://github.com/cognitedata/cognite-sdk-js/pull/1275 (I've notified the author about it). However, it's blocked by another SDK PR. Seeing that this PR is part of our goals for this sprint (end-of-week), could we rather migrate to this filter definition later on? It seems unreasonable to block this feature on two barely related PRs